en/overview/proj4_overview.rst

```diff
@@ -3,10 +3,16 @@
 :Version: osgeo-live11.0
 :License: Creative Commons Attribution 3.0 Unported (CC BY 3.0)
 
-.. image:: ../../images/project_logos/logo-proj4.png
+.. image:: /images/project_logos/logo-proj4.png
   :alt: project logo
   :align: right
```

Comment:
I am not sure if PROJ is an osgeo project or not.

* https://github.com/OSGeo/proj.4
* http://proj4.org

So yes, it is part of MetaCRS so it is a community project.